browser/browser/browser.ftl

```diff
@@ -1323,3 +1325,128 @@ onboarding-checklist-button-label = Finish setup
 onboarding-aw-finish-setup-button =
     .label = Finish setup
     .tooltiptext = Finish setting up { -brand-short-name }
+
+## The urlbar trust panel
+
+trustpanel-etp-label-enabled = Enhanced Tracking Protection is on
+trustpanel-etp-label-disabled = Enhanced Tracking Protection is off
+trustpanel-etp-description-enabled = If something looks broken on this site, try turning off protections.
+trustpanel-etp-description-disabled = { -brand-product-name } thinks companies should follow you less. We block as many trackers as we can when you turn on protections.
+trustpanel-connection-label-secure = Connection secure
+trustpanel-connection-label-insecure = Connection not secure
+trustpanel-header-enabled = { -brand-product-name } is on guard
+trustpanel-description-enabled = You’re protected. If we spot something, we’ll let you know
+trustpanel-header-disabled = You turned off protections
+trustpanel-description-disabled = { -brand-product-name } is off-duty. We suggest turning protections back on.
+trustpanel-clear-cookies-button = Clear cookies and site data
+trustpanel-privacy-link = Privacy Settings
+# Variables
+#  $host (String): the hostname of the site that is being displayed.
+trustpanel-clear-cookies-header =
+    .title = Clear cookies and site data for { $host }
+trustpanel-clear-cookies-description = Removing cookies and site data might log you out of websites and clear shopping carts.
+trustpanel-clear-cookies-subview-button-clear = Clear
+trustpanel-clear-cookies-subview-button-cancel = Cancel
+# Variables
+#  $host (String): the hostname of the site that is being displayed.
+trustpanel-site-information-header =
+    .title = Connection protections for { $host }
+trustpanel-connection-secure = You are securely connected to this site
+trustpanel-connection-not-secure = You are not securely connected to this site.
```

Comment:
Should this end with a period or not (string above)?

Reply:
Or other way around… 😓 

Still landing in https://phabricator.services.mozilla.com/D258025?vs=1093375&id=1093478

Reply:
Thanks, I'll push strings once I see that lands